crosspoint_reader/driver.py

```
@@ -274,6 +276,41 @@ def _ensure_dir(self, parent_path, subdirs):
+            )
+            
+            # Create temp file for converted EPUB
+            temp_file = PersistentTemporaryFile(suffix='_baseline.epub')
+            temp_path = temp_file.name
+            temp_file.close()
+            
+            # Convert
+            self._log(f'[CrossPoint] Converting: {os.path.basename(input_path)}')
+            converter.convert_epub(input_path, temp_path)
+            
+            return temp_path
```

Ruff (0.15.2)

[warning] Consider moving this statement to an `else` block

(TRY300)